tests/postgres_tests/test_aggregates.py

```diff
@@ -164,6 +167,10 @@ def test_string_agg_charfield_ordering(self):
             (F('char_field').desc(), 'Foo4;Foo3;Foo2;Foo1'),
             (F('char_field').asc(), 'Foo1;Foo2;Foo3;Foo4'),
             (F('char_field'), 'Foo1;Foo2;Foo3;Foo4'),
+            ('char_field', 'Foo1;Foo2;Foo3;Foo4'),
+            ('-char_field', 'Foo4;Foo3;Foo2;Foo1'),
```

Comment:
These two tests are not related with this fix, please move them to a separate commit.

Reply:
Done!